sqlalchemy_exasol/pyodbc.py

```diff
@@ -164,5 +167,139 @@ def is_disconnect(self, e, connection, cursor):
 
         return super().is_disconnect(e, connection, cursor)
 
+    @staticmethod
+    def _is_sql_fallback_requested(**kwargs):
+        is_fallback_requested = kwargs.get("use_sql_fallback", False)
+        if is_fallback_requested:
+            logger.warning("Using sql fallback instead of odbc functions")
+        return is_fallback_requested
+
+    @reflection.cache
+    def _tables_for_schema(self, connection, schema, table_type=None, table_name=None):
+        schema = self._get_schema_for_input_or_current(connection, schema)
+        table_name = self.denormalize_name(table_name)
+        conn = connection.engine.raw_connection()
```

Comment:
is this engine attribute garantueed?

Reply:
From what I have seen in the docs and the code, `yes`.